docker/netbox/configuration/plugins.py

```diff
@@ -6,9 +6,18 @@
 
 PLUGINS = ["netbox_diode_plugin"]
 
-PLUGINS_CONFIG = {
-    "netbox_diode_plugin": {
-        "diode_target": "grpc://localhost:8080/diode", # The Diode gRPC target for communication with Diode server, default: "grpc://localhost:8080/diode"
-        "disallow_diode_target_override": False, # Disallow the Diode target to be overridden by the user, default: False
-    }
-}
+# PLUGINS_CONFIG = {
+#     "netbox_diode_plugin": {
```

Comment:
Should it be commented out?

Reply:
This config is for development docker purposes and plugin uses its default settings.